core/tests/protractor_desktop/userJourneys.js

```diff
@@ -32,6 +32,9 @@ var users = require('../protractor_utils/users.js');
 var waitFor = require('../protractor_utils/waitFor.js');
 var workflow = require('../protractor_utils/workflow.js');
 
+var moderatorPage = element(by.css('.protractor-test-moderator-page'));
+var errorContainer = element(by.css('.protractor-test-error-container'));
```

Comment:
We should define elements and actionable functions in page object file i.e, protractor_utils/*.js file

Reply:
done